xml/vk.xml

```diff
@@ -11223,5 +11223,11 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="&quot;VK_EXT_extension_285&quot;"              name="VK_EXT_extension_285"/>
             </require>
         </extension>
+        <extension name="VK_EXT_extension_286" number="286" type="instance" author="MESA" contact="Drew DeVault <sir@cmpwn.com>" supported="disabled">
```

Comment:
Shouldn't it have `MESA` prefix then? `MESA` seem defined in XML as tag.

Reply:
I think EXT is appropriate and MESA is not. I wrote that before I understood the meaning of the field and never went back to change it. Just pushed a fixed version now.

Reply:
It also really doesn't matter when just reserving an extension number.  We can do "the right thing" when the real version of the extension rolls out.

Reply:
Well yea, though it might be annoying to refactor the whole extension spec with proper tag.